src/formio/templates/text.ejs

```diff
@@ -16,6 +16,7 @@
   {{attr}}="{{ctx.input.attr[attr]}}"
   {% } %}
   id="{{ctx.instance.id}}-{{ctx.component.key}}"
+  aria-labelledby="label-{{ctx.instance.id}}-{{ctx.component.key}} {{ctx.instance.id}}-{{ctx.component.key}} suffix-{{ctx.instance.id}}-{{ctx.component.key}}"
```

Comment:
I think we should only do this when there is a suffix, this template is also rendered for a `textfield` component which doesn't have suffixes. So the suffix `<span>` will not exist, and the ID therefor will not exist, so these would be broken references. I also think if there's no suffix, it doesn't make sense to use a different label than the default.